draft-ietf-taps-interface.md

```diff
@@ -1049,26 +1048,47 @@ Type:
 : Enumeration
 
 Default:
-: Disabled
+: Disabled for connections created through initiate and rendezvous, Passive for listeners
```

Comment:
I afraid I don't fully understand this. In case of MPTCP does disabled mean I will not open any new sub flows, reject all sub flows (SYNs) from the other end, or not even include the MPTCP option in the handshake of the initial connection?

Reply:
Or let me put this differently: if multi path support is enabled and a MP option sent in the initial handshake, I think we only need "active" and "passive" as modes because otherwise if I reject all your SYNs why should I even include the MPTCP option in the initial handshake?

Reply:
We need Disabled here:
- Disabled results in not sending the MPTCP option, 
- Passive results in sending the MPTCP option it but not opening additional subflows.
- Active results in sending the MPTCP option and opening additional subflows

Reply:
Still not sure here because the decision to open a new subflows really depends on the path/interface. Not sure you even want to say: whenever you see a new IP address, immediately open a new subflow. So I think you only want disable and enable here and leave the decision to open new subflows on the interface level.

Reply:
I agree that we should have some control on the Interface level, but relying on this also for the active/passive path establishment looks to over-complicate things for me.
This Property just controls whether the local side accepts additional incoming sub-flows or whether it also actively tries to establish them. I suspect that this one most applications can easily set:
- Passive in the Server-Case
- Active or Disabled in the Client case
The per-interface one is a major hassle and I believe most programmer really want the implementation to take care here.

What we should discuss is whether we want to make "Passive" also the default for connections created through initiate and rendezvous…